doc/reference/reference_lua/box_stat/vinyl.rst

```diff
@@ -190,4 +158,45 @@ for dumping or compaction:
 
   A dump will also occur during a
   :doc:`snapshot </reference/reference_lua/box_snapshot>` operation.
-  
\ No newline at end of file
+
+.. _box_introspection-box_stat_vinyl_memory_tuple:
+
+box.stat.vinyl().memory.tuple
```

Comment:
There's the `box.stat.vinyl().memory` section above. The new counter description should probably be added to it.

Reply:
Fair. Done as suggested.

Reply:
Oops, I missed that we already had that memory section in stats. Thanks!